atd-toolbox/data_model/python/sql_templates/fatalities_view.sql

```diff
@@ -0,0 +1,48 @@
+drop table fatalities cascade;
+
+drop view if exists fatalities_view cascade;
+
+create or replace view fatalities_view AS
+select 
+people.id as person_id,
+crashes.id as crash_id,
+crashes.crash_id as cris_crash_id,
+units.id as unit_id,
+CONCAT_WS(' ', people.prsn_first_name, people.prsn_mid_name, people.prsn_last_name) as victim_name,
+TO_CHAR(crashes.crash_timestamp at time zone 'US/Central', 'yyyy') AS year,
+CONCAT_WS(' ',
+  crashes.rpt_block_num,
+  crashes.rpt_street_pfx,
+  crashes.rpt_street_name,
+  '(',
+  crashes.rpt_sec_block_num,
+  crashes.rpt_sec_street_pfx,
+  crashes.rpt_sec_street_name,
+  ')') AS location,
+to_char(
+  public.crashes.crash_timestamp at time zone 'US/Central', 'YYYY-MM-DD'
+  ) as crash_date_ct,
+to_char(
+  public.crashes.crash_timestamp at time zone 'US/Central', 'HH24:MI:SS'
+  ) as crash_time_ct,
+-- get ytd fatality, partition by year and sort by date timestamp
+ROW_NUMBER() OVER (
+    PARTITION BY EXTRACT(year FROM crashes.crash_timestamp) 
+    ORDER BY crashes.crash_timestamp ASC) 
+    AS ytd_fatality,
+-- get ytd fatal crash, partition by year and sort by date timestamp.
+-- records with the same crash_id will get "tie" rankings thus making
+-- this column count each crash rather than each fatality
+DENSE_RANK() OVER (
+    PARTITION BY EXTRACT(year FROM crashes.crash_timestamp) 
+    ORDER BY crashes.crash_timestamp ASC, crashes.crash_id) 
+    AS ytd_fatal_crash,
+crashes.case_id,
+crashes.law_enforcement_fatality_num,
```

Comment:
still think `law_enforcement_fatality_num` is a kinda confusing name for the column bc its a ytd fatal crash number not fatality i.e its counting crashes not persons. but the vz team knows what it means so :shrug: 

`law_enforcement_fatal_crash_num`?

Reply:
i fixed this in https://github.com/cityofaustin/atd-vz-data/pull/1488!

Reply:
don't sweat it here—i'll resolve conflicts. there are a decent number at this point